src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -111,19 +111,19 @@ private String generateRecipeContent(Recipe recipe) {
     /**
      * Retrieves the {@code Recipe} with the same name as the specified name
      * from a given list of recipes.
-     * Returns null if a recipe with the same name cannot be found.
+     * Throws a CommandException if a recipe with the same name cannot be found.
      *
      * @param lastShownList the list of recipes to search from.
      * @param recipeName the name of the recipe to view.
-     * @return the recipe from the list matching the specified name.
```

Comment:
Is this line of java docs unintentionally deleted too?

Reply:
Yep, line has been re-added